templates/default/flyingduck.hcl.erb

```diff
@@ -14,5 +14,13 @@ services = [
             timeout = "20s"
             status = "passing"
         }
+    },
+    {
+        id = "flyingduck/server"
+        name = "flyingduck"
+        tags = [
+            "server"
+        ]
+        port = <%= node['flyingduck']['port'] %>
```

Comment:
Can you add the health check also for this port? You can re-use the same check for the monitoring port. This is an example:
https://github.com/logicalclocks/hops-hadoop-chef/blob/f130bea4200d88c1c53c274058c22617b51f41a4/templates/default/consul/nn-consul.hcl.erb#L25